catalog/core/catalog-email-impl/src/main/java/org/codice/alliance/core/email/impl/EmailSenderImpl.java

```diff
@@ -0,0 +1,188 @@
+            InternetAddress emailAddr = new InternetAddress(toEmail);
+            emailAddr.validate();
+
+            Properties properties = createSessionProperties();
+
+            Session session = Session.getDefaultInstance(properties);
+
+            MimeMessage mimeMessage = new MimeMessage(session);
+            mimeMessage.setFrom(new InternetAddress(fromEmail));
+            mimeMessage.addRecipient(Message.RecipientType.TO, emailAddr);
+            mimeMessage.setSubject(subject);
+
+            BodyPart messageBodyPart = new MimeBodyPart();
+            messageBodyPart.setText(body);
+
+            Multipart multipart = new MimeMultipart();
+            multipart.addBodyPart(messageBodyPart);
+
+            Holder<Long> bytesWritten = new Holder<>(0L);
+
+            for (Pair<String, InputStream> attachment : attachments) {
+
+                messageBodyPart = new MimeBodyPart();
+                File file = File.createTempFile("email-sender-", ".dat");
+                tempFiles.add(file);
+
+                copyDataToTempFile(file, attachment.getValue(), bytesWritten);
+                messageBodyPart.setDataHandler(new DataHandler(new FileDataSource(file)));
+                messageBodyPart.setFileName(file.getAbsolutePath());
+                multipart.addBodyPart(messageBodyPart);
+
+            }
+
+            mimeMessage.setContent(multipart);
+
+            Transport.send(mimeMessage);
+
+            LOGGER.debug("Email sent to " + toEmail);
+
+        } catch (AddressException e) {
+            throw new IOException("invalid email address: email=" + toEmail, e);
+        } catch (MessagingException e) {
+            throw new IOException("message error occured on send", e);
+        } finally {
+            tempFiles.forEach(file -> {
+                if (!file.delete()) {
+                    LOGGER.debug("unable to delete tmp file: path={}", file);
+                }
+            });
+        }
+    }
+
+    private void copyDataToTempFile(File file, InputStream inputStream, Holder<Long> bytesWritten)
+            throws IOException {
+
+        try (OutputStream outputStream = new FileOutputStream(file)) {
+
+            byte[] buffer = new byte[BUFFER_SIZE];
+
+            int c;
+            while ((c = inputStream.read(buffer)) != -1) {
+                outputStream.write(buffer, 0, c);
+                bytesWritten.value = bytesWritten.value + c;
+                if (bytesWritten.value > maxAttachmentSize) {
+                    throw new IOException(
+                            "total attachment size exceeds limit: limit=" + maxAttachmentSize);
+                }
+            }
+
+        }
+
+    }
+
+    private Properties createSessionProperties() {
+        Properties properties = System.getProperties();
+        properties.setProperty(SMTP_HOST_PROPERTY, mailHost);
+        properties.setProperty(SMTP_AUTH_PROPERTY, "false");
+        properties.setProperty(SMTP_PORT_PROPERTY, Integer.toString(mailPort));
```

Comment:
I haven't need to set that with my testing, but I can look into it. Are you thinking it should be set to TRUE or to FALSE?